docs/_plugins/rhds.cjs

```diff
@@ -153,7 +132,42 @@ module.exports = function(eleventyConfig, { tagsToAlphabetize }) {
 
   eleventyConfig.addTransform('demo-subresources', demoPaths);
 
-  eleventyConfig.addTransform('demo-lightdom-css', lightdomCss);
+  eleventyConfig.addTransform('demo-lightdom-css', async function(content) {
+    const { outputPath, inputPath } = this;
+    const { pfeconfig } = eleventyConfig?.globalData ?? {};
+    const { aliases } = pfeconfig;
+
+    if (inputPath === './docs/elements/demos.html' ) {
+      const tagNameMatch = outputPath.match(/\/elements\/(?<tagName>[-\w]+)\/demo\//);
+      if (tagNameMatch) {
+        const { tagName } = tagNameMatch.groups;
+
+        // slugify the value of each key in aliases creating a new cloned copy
+        const modifiedAliases = { ...aliases };
+        for (const key in aliases) {
```

Comment:
syntax nit, for these kinds of things, `fromEntries` can be preferred, as it's terser, doesn't require the hasOwnProperty call, and focuses the reader's attention on the transformation in the mapping function. it also returns a new object, which let's us skip the spread in the previous line and encourages us not to mutate the inputs, which cuts down on mental execution

```js
const slugifiedAliases = Object.fromEntries(Object.entries(aliases).map(([key, value]) =>
  [slugify(key, { strict: true, lower: true }), value]));
```